src/callbacks/callbacks.jl

```diff
@@ -524,3 +526,16 @@ function gc_func(integrator)
     )
     return nothing
 end
+
+
+function update_surface_temp!(integrator)
+    d = 10 # ocean mixed layer depth [m]
+    ρₒ = 1020 # ocean density [kg / m³]
```

Comment:
```suggestion
    ρ_ocean = 1020 # ocean density [kg / m³]
```
(I prefer not to use `O` in subscript or superscript because I find it hard to differ it from `0`.)